Functions.Templates/Templates/AuthenticationEventsTrigger-CSharp/AuthenticationEventsTriggerCSharp.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart.Actions;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Company.Function
+{
+    /// <summary>Example functions for token augmentation</summary>
+    public static class AuthenticationEventFunctions
```

Comment:
The class name should be AuthenticationEventsTriggerCSharp

Reply:
Done